src/Client.php

```diff
@@ -294,7 +293,7 @@ public function listDatabaseNames(array $options = []): Iterator
      * List databases.
      *
      * @see ListDatabases::__construct() for supported options
-     * @return DatabaseInfoIterator
+     * @return Iterator
```

Comment:
Can't you use the generics syntax here? Otherwise this change is a regression for typing.
```suggestion
     * @return Iterator<DatabaseInfo>
```